src/pybind/mgr/dashboard/module.py

```diff
@@ -976,14 +995,15 @@ def index(self):
                 }
 
                 return template.render(
+                    url_prefix = global_instance().url_prefix,
                     ceph_version=global_instance().version,
                     path_info='/osd' + cherrypy.request.path_info,
                     toplevel_data=json.dumps(toplevel_data, indent=2),
                     content_data=json.dumps(content_data, indent=2)
                 )
 
-        cherrypy.tree.mount(Root(), "/", conf)
```

Comment:
Oh, never mind, I guess this is assuming that the reverse proxy is configured to point to "/foo" rather than pointing to "/", I think I get it now.